pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "pydantic-settings >=2, <3.0",
     "quantinuum-schemas >=3, <4.0",
     "hugr >=0.11.5, <1.0.0",
+    "pytket-qir>=0.22.0",
```

Comment:
I'm worried slightly about the number of dependencies are pulling in for this project, would having `pyqir` as an optional dependency be more appropriate.